scavenge/tutorial/01-background.md

```diff
@@ -4,23 +4,19 @@ order: 1
+This tutorial was first presented as a workshop at GODays 2020 Berlin by [Billy Rennekamp](https://twitter.com/billyrennekamp).
 
 The goal of this session is to get you thinking about what is possible when developing applications that have access to **digital scarcity as a primitive**. The easiest way to think of scarcity is as money; If money grew on trees it would stop being _scarce_ and stop having value. We have a long history of software which deals with money, but it's never been a first class citizen in the programming environment. Instead, money has always been represented as a number or a float, and it has been up to a third party merchant service or some other process of exchange where the _representation_ of money is swapped for actual cash. If money were a primitive in a software environment, it would allow for **real economies to exist within games and applications**, taking one step further in erasing the line between games, life and play.
 
-We will be working today with a Golang framework called the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk). This framework makes it easy to build **deterministic state machines**. A state machine is simply an application that has a state and explicit functions for updating that state. You can think of a light bulb and a light switch as a kind of state machine: the state of the "application" is either `light on` or `light off`. There is one function in this state machine: `flip switch`. Every time you trigger `flip switch` the state of the application goes from `light on` to `light off` or vice versa. Simple, right?
+We will be working today with a Go framework called the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk). This framework makes it easy to build **deterministic state machines**. A state machine is simply an application that has a state and explicit functions for updating that state. You can think of a light bulb and a light switch as a kind of state machine: the state of the "application" is either `light on` or `light off`. There is one function in this state machine: `flip switch`. Every time you trigger `flip switch` the state of the application goes from `light on` to `light off` or vice versa. Simple, right?
 
-<img src="./img/light-bulb.png" style="margin:auto; display:block">
+A **deterministic** state machine is just a state machine in which an accumulation of actions, taken together and replayed, will have the same outcome. So if we were to take all the `switch on` and `switch off` actions of the entire month of January for some room and replay then in August, we should have the same final state of `light on` or `light off`. There's should be nothing about January or August that changes the outcome (of course a _real_ room might not be deterministic if there were things like power shortages or maintenance that took place during those periods).
```

Comment:
```suggestion
A **deterministic** state machine is just a state machine in which an accumulation of actions, taken together and replayed, will have the same outcome. So if we were to take all the `switch on` and `switch off` actions of the entire month of January for some room and replay them in August, we should have the same final state of `light on` or `light off`. There should be nothing about January or August that changes the outcome (of course a _real_ room might not be deterministic if there were things like power shortages or maintenance that took place during those periods).
```

Reply:
u@fadeev I really appreciate this explanation, how did we live without it? thank you